### PR TITLE
Create dandelion_fr.md

### PR DESCRIPTION
**Pull request by Hsterofemia**
French translation of Dandelion_en

### CODE REVIEW

File: monero-outreach-docs/translations/fr/archive/dandelion_fr.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+Sa structure comprend des milliers de nœuds répartis de par la planète — des dizaines de milliers si on compte les nœuds passifs — chacun connecté, via Internet, habituellement à un petit groupe d'autres nœuds appelés pairs [1]. Les nœuds communiquent avec leurs pairs en utilisant le même protocole TCP ("Transmission Control Protocol" - Protocole de Contrôle de Transmission) utilisé par les fureteurs et les serveurs Web. Les pairs d'un nœud paraissent aléatoires et n'ont pas besoin de rapprochement géographique. Un nombre commun de pairs est huit, mais certains nœuds en ont des centaines. Chaque nœud trouve ses pairs en envoyant d'abord une requête à un nœud maître pour sa liste de pairs, pour ensuite envoyer la même requête aux pairs de cette liste, et ainsi de suite. Le résultat est un réseau de plusieurs ordinateurs, chacun communicant par TCP avec au mieux une visibilité partielle et limitée du réseau entier.
```
Comment (Tomahna81) [inline monero-outreach-docs/translations/fr/archive/dandelion_fr.md:11]: fureteurs >>> navigateurs  

Reason: reviewer choice, but this is certainly due to a regional difference. People native from France - as opposed to e.g Quebec - might have never heard the term 'fureteur'

File: monero-outreach-docs/translations/fr/archive/dandelion_fr.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+Ces nœuds agissent comme un portail pour permettre au portefeuilles de communiquer avec le réseau P2P. Certains portefeuilles, comme le portefeuille GUI officiel, peuvent exécuter leur propre nœud, tandis que d'autres, comme MyMonero sur un téléphone intelligent, utilisent toujours un nœud distant. Les portefeuilles dépensent Monero en créant et diffusant une transaction via son nœud portail, afin que la transaction trouve ultimement son chemin vers le nœud d'un mineur pour être incluse dans un bloc miné. Chaque transaction comporte une quantité limitée d'informations, mais qui contient malgré tout des données significatives, incluant un ensemble de sources possibles des fonds dépensés.
```
Comment (Tomahna81) [inline monero-outreach-docs/translations/fr/archive/dandelion_fr.md:13]: téléphone intelligent >>> téléphone intelligent (smartphone) 

Reason: The literal translation may surprise some French speakers of specific regions.

File: monero-outreach-docs/translations/fr/archive/dandelion_fr.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+### Dandelion
+
+Afin d'adresser ce problème, une équipe de chercheurs de l'Université de l'Illinois ont d'abord développé un ensemble de techniques qu'ils ont appelé Dandelion [2]. L'application devait être pour Bitcoin, mais Dandelion s'applique également à Monero. Le principe de Dandelion est de de router, en premier lieu, les transactions à un nœud distant d'une manière spéciale et indétectable avant d'initier la propagation par "flooding".
```
Comment (Tomahna81) [inline monero-outreach-docs/translations/fr/archive/dandelion_fr.md:25]: Dandelion est de de router >>> Dandelion est de router  

Reason: Typo

File: monero-outreach-docs/translations/fr/archive/dandelion_fr.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+Pour adresser cette insuffisance, les auteurs ont proposé Dandelion. Dandelion définit un procédé pour trouver un nœud proxy pour effectuer la transmission, qu'on appelle la phase d'anonymité (ou la phase tige - "stem" en anglais). L'outil établit un second procédé pour l'émission, appelé la phase ouate ("spread" ou "fluff" en anglais). En général, les deux phases font appel à un ensemble de pairs différent, avec la différence marquée que l'ensemble de connexions pour la phase d'anonymité change avec le temps. Le nom Dandelion nous vient du procédé où, dans un premier temps, le système trouve un nœud proxy selon un chemin de recherche particulier et linéaire, puis à partir de ce nœud proxy le système propage l'information rapidement et symétriquement-la forme de la propagation d'information ressemble à une fleur de pissenlit ("Dandelion" en anglais).
```
Comment (Tomahna81) [inline monero-outreach-docs/translations/fr/archive/dandelion_fr.md:29]: [1] 
Dandelion définit un procédé pour trouver un nœud proxy pour effectuer la transmission, qu'on appelle la phase d'anonymité (ou la phase tige - "stem" en anglais)  
>>>  
Dandelion définit un procédé qu'on appelle la phase d'anonymité (ou la phase tige - "stem" en anglais), afin de trouver un nœud proxy qui effectuera la transmission   

Reason: Reordering and slightly modifying the wording makes it clearer to me

[2]
appelé la phase ouate ("spread" ou "fluff" en anglais)  >>> appelé la phase de dispersion ("spread" ou "fluff" en anglais)   

Reason: I discovered the word 'ouate' by reading this. I think it is indeed a possible translation. But it is I think quite rare (or is specific to some regions?). Therefore, I am not sure that this is the most appropriate term. A more common word, such as 'dispersion' or an adequate translation of 'fluff' (phase duvet? phase coton?) is probably preferable.
Note that this expression 'phase ouate' appears at several occasion over the text. I noted this recommendation only here (once), but obviously whatever is the choice of the translator (keep 'ouate' or change it) the potential modification will need to be propagated all along the text for sake of consistency.

File: monero-outreach-docs/translations/fr/archive/dandelion_fr.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+Il y a un élément additionnel et complémentaire de Dandelion++ appelé mécanisme à sûreté intégrée. Chaque nœud qui relaye une transaction pendant la phase tige démarre un décompte pour cette transaction. Si un lapse de temps prédéterminé s'écoule sans que le nœud ne reçoive la même transaction en retour pendant la phase ouate, il démarre sa propre phase ouate. Ce mécanisme sert deux objectifs: il frustre toute tentative de défaire l'anonymité en utilisant la chronologie des transmission, et il empêche les attaques dites trou noir où des nœuds malveillants détruisent les transactions pendant la phase tige plutôt que les relayer.
+
+Grâce à ces techniques, Dandelion++ fournit une garantie formelle de résistance aux attaques à l'anonymité du réseau. Cette fin est atteinte en utilisant des techniques locale, contrairement au réseau Tor qui s'appuie un raisonnement global. Chaque nœud Dandelion++ se comporte selon sa propre logique interne. Le résultat est une technique rapide, efficiente et efficace pour empêcher l'association d'adresses IP aux transaction par les acteurs les plus sophistiqués.
```
Comment (Tomahna81) [inline monero-outreach-docs/translations/fr/archive/dandelion_fr.md:45]: des techniques locale >>> des techniques locales  

Reason : Typo

**Review by Tomahna81 — CHANGES_REQUESTED** [review]
Few minor issues. Overall good translation. Good Job!

File: monero-outreach-docs/translations/fr/archive/dandelion_fr.md (hunk 1 above)
Comment (Hsterofemia) [reply monero-outreach-docs/translations/fr/archive/dandelion_fr.md:11]: Integrated.

File: monero-outreach-docs/translations/fr/archive/dandelion_fr.md (hunk 2 above)
Comment (Hsterofemia) [reply monero-outreach-docs/translations/fr/archive/dandelion_fr.md:13]: Good point, Integrated.

File: monero-outreach-docs/translations/fr/archive/dandelion_fr.md (hunk 3 above)
Comment (Hsterofemia) [reply monero-outreach-docs/translations/fr/archive/dandelion_fr.md:25]: Integrated

File: monero-outreach-docs/translations/fr/archive/dandelion_fr.md (hunk 4 above)
Comment (Hsterofemia) [reply monero-outreach-docs/translations/fr/archive/dandelion_fr.md:29]: I had trouble with this one, and I agree with your recommendation. The analogy in the original text is obviously to the dandelion flower, with a stem and the fluff part that blows away in the wind. The French term for this is indeed ouate, which is why I chose it, but "phase ouate" isn't quite to my liking either. I'm going to integrate your recommendation of "phase de dispersion".

File: monero-outreach-docs/translations/fr/archive/dandelion_fr.md (hunk 5 above)
Comment (Hsterofemia) [reply monero-outreach-docs/translations/fr/archive/dandelion_fr.md:45]: Integrated

**Comment by Hsterofemia** [comment]
Integrated all reviewer's recommendations. Changed all instances of "phase ouate" to "phase de dispersion". Ready for final approval.

**Review by Tomahna81 — APPROVED** [review]

**Review by lh1008 — APPROVED** [review]
Approved.

Thank you @Hsterofemia and @Tomahna81 

Great work!

**Pull request merged**